docs/handbook/image-file-formats.rst

```diff
@@ -1124,6 +1124,11 @@ The :py:meth:`~PIL.Image.Image.save` method supports the following options:
 **method**
     Quality/speed trade-off (0=fast, 6=slower-better). Defaults to 4.
 
+**exact**
+    If true, preserve the transparent RGB values. Otherwise, discard
+    invisible RGB values for better compression. Defaults to false.
+    Requires LibWebP 0.5.0 or later.
```

Comment:
```suggestion
    Requires libwepb 0.5.0 or later.
```
While it is "WebP", a quick Google leads me to believe it is "libwebp". See https://en.wikipedia.org/wiki/WebP

Reply:
Yeah, you're right. Done.